### ISSUE TITLE
Spybot S&D Bootable CD

### ISSUE THREAD

**Issue by Tobias-B-Besemer** [ScriptTiger/Windows-911#3]
It's not really a Antivirus, so I prefer to make an issue for it... ;-)

Here is a how-to, how the customers can create there own S&D Boot-CDs:
https://forums.spybot.info/showthread.php?56353-How-to-create-a-Boot-CD-with-Spybot

**Comment by ScriptTiger** [comment]
I'll keep this issue open for now, but UBCD also comes with SUPERAntiSpyware. Maybe we can figure out how to put it in later as a preference thing, like the AV disks.

**Comment by Tobias-B-Besemer** [comment]
Hi ScriptTiger,
there is one feature of S&D I think you should really mention in your documentation...
...because IMHO the really cool thing on S&D is, that it scans your registry and cleans it also up - what many AV-Solutions AFAIK don't do...
And the S&D-CD have the benefit, that it scans your HDs for installed OSes (Partitions) and then checks each Registry its finds... ;-)
IMHA a really cool feature! :-)
I also think, you can add it to the AV-Solution (e.g. as last one), because S&D have also a AV-Solution integrated (for those who pay), and other AV-Solutions delete (in any way) Spyware... So for the normal customer the different is IMHO not so big - he just needs to know, that it is better (suggested by 'Professionals' ^^), to work with both solutions afterwards, to clean up all...
Greets, Tobias.

**Comment by ScriptTiger** [comment]
Hmmm... Can you make a PR for the "Other" section for now and consolidate all those points into a 1 paragraph description? Of if you can think of another good spyware scanner disk, we can make a section for it. Scanning registries in multiple partitions does sound pretty cool, but I don't think it's very common to have more than one Windows partition on a system, usually there's just one and maybe a few special-purpose Linux distros and a Windows recovery partition. Unless we're talking about VMs, scanning multiple registries in that case might be pretty awesome.

**Comment by Tobias-B-Besemer** [comment]
Yes, normally one Windows is more then enough for a PC... ;-) :D
I just explained it this way...you can also say: It finds the right partition on your HD, no matter how you partitioned it, finds the registry, scans and cleans it up, without any need for the user to interact. ;-)

I will make a PR... But I'm a German and English is not my mother-language... So you maybe have to fix some things later by yourself... ;-)

I normally don't fix anymore PCs! All the people that asked me in the past wasn't able to understand how many years of learning are in this computer-thing and thought, that they make me happy, if they give me e.g. 10 Dollars a hour... :D
So it's really good, that there exist now one more documentation (yours) on at least how to build a repair system... :D
But in the days I have fixed PCs, I used always Avira and S&D...
...but if you want to have something more in your list, so that the user have a choice, I would suggest Malwarebyte.
(I did ATM a search about 'Ransomware Boot CDs', but there comes also only the same stuff up...)
Malwarebyte have still no Boot CD!
But here is the link to the support-doc describing where to download the MBAR-Tool that you should install on the infected system:
https://support.malwarebytes.com/docs/DOC-1089

**Comment by ScriptTiger** [comment]
@Tobias-B-Besemer, I have to apologize to you, I have been reading over the UBCD list again and it doesn't include any spyware scanner, I was thinking about UBCD4Win, a different project that is dead but there is still a download available here:
http://www.majorgeeks.com/files/details/ubcd4win.html
SUPERAntiSpyware doesn't offer a boot disk by itself, so Spybot S&D will definitely be useful to cover that gap.

I totally forgot about UBCD4Win because its project's website and forums and official documentation have all disappeared from the Internet, but it was another disk I definitely used to use. I will have to do some digging on archive.org to find out everything that's in it. I do have the disk, but I'm not going to take notes on all of the stuff in there because there is a lot.

**Comment by Tobias-B-Besemer** [comment]
Here is a other link to the Anti-Rootkit-Tool:
https://www.malwarebytes.com/antirootkit/

If you like, I can try to make a PR for it...

**Comment by ScriptTiger** [comment]
For sure, go for it! Just keep adding them to the "other" for now, and when we get 2 or more in a category, we will make a new section.

**Comment by ScriptTiger** [comment]
I posted to Reddit about a possible revival for UBCD4Win and there are already a couple links to communities that might be interested in it. I don't know if I personally have the time to work on it much, but I really do think it's an awesome idea.
https://www.reddit.com/r/computertechs/comments/8odynq/ubcd4win_revival/

**Comment by Tobias-B-Besemer** [comment]
I guess I try to make the 'big hit' and make you a PR for suggestion on a 'Anti-Add- & Spyware'-Topic... ;-)

Had a look on reddit and a short look on the UBCD4Win-Achive...
I never used it... As I already said, there was the original Hiren's for me... ^^
But looks great! And if there is interest to revival it, I will try to spend some time to help! :-)
